net/nmap/Portfile

```diff
@@ -31,6 +32,10 @@ checksums      rmd160  8418fcef96848a6ae6bbbeee935b34ce047eab5e \
                sha256  d71be189eec43d7e099bac8571509d316c4577ca79491832ac3e1217bc8f92cc \
                size    11102195
 
+# see https://trac.macports.org/ticket/68543
+patchfiles  nmap-7.94-fix-32bit.patch \
+            gcc-4.2-workaround.patch
```

Comment:
I'd still prefer the gcc-4.2 patch to be only applied in platform blocks that restrict it to 10.5 and earlier. Hopefully upstream will include the 32bit patch (or equivalent) and we can just keep this patching only on older hosts that actually need it.

Reply:
gcc-4.2 is a part of Xcode on 10.6 too, so any patches targeting it should extend to 10.6.